src/Raven.Server/Documents/Indexes/Persistence/Corax/CoraxWriterScope.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Corax;
+using Raven.Server.Documents.Indexes.Persistence.Corax.Enumerators;
+
+namespace Raven.Server.Documents.Indexes.Persistence.Corax
+{
+    public class CoraxWriterScope
```

Comment:
Discussed that with @maciejaszyk 
We will handle that a bit differently to simply the code in the converter.